src/tests/useTaskManager.test.ts

```diff
@@ -0,0 +1,71 @@
+import { describe, it } from 'vitest';
+import { act, renderHook, waitFor } from '@testing-library/react';
+import { useTaskManager } from '../hooks/useTaskManager';
+import { mockPosts } from './mocks/handlers';
+
```

Comment:
✨ Suggestion:  This test should be `colocated` with the `useTaskManager.ts` hook; they should be in the same directory.

Reply:
moved to same location